optaplanner-core/src/main/java/org/optaplanner/core/api/solver/manager/SolverManager.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.solver.manager;
+
+import java.util.concurrent.ThreadFactory;
+import java.util.function.Consumer;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.solver.manager.DefaultSolverManager;
+
+public interface SolverManager<Solution_> extends AutoCloseable {
+
+    static <Solution_> SolverManager<Solution_> createFromXmlResource(String solverConfigResource) {
+        return DefaultSolverManager.createFromXmlResource(solverConfigResource);
+    }
+
+    static <Solution_> SolverManager<Solution_> createFromXmlResource(String solverConfigResource, ClassLoader classLoader) {
+        return DefaultSolverManager.createFromXmlResource(solverConfigResource, classLoader);
+    }
+
+    static <Solution_> SolverManager<Solution_> createFromXmlResource(String solverConfigResource, ThreadFactory threadFactory) {
+        return DefaultSolverManager.createFromXmlResource(solverConfigResource, threadFactory);
+    }
+
+    static <Solution_> SolverManager<Solution_> createFromXmlResource(String solverConfigResource,
+                                                                      ClassLoader classLoader, ThreadFactory threadFactory) {
```

Comment:
As far as I know, we don't align code on columns. Please configure your IDE code style properly.
(Yes, we do not respect the KIE code style in this particular regard.)

Reply:
My $0.02. Format it like this:
```java
    static <Solution_> SolverManager<Solution_> createFromXmlResource(
            String solverConfigResource,
            ClassLoader classLoader,
            ThreadFactory threadFactory
    ) {
```
- doesn't conflict with KIE conventions' alignment style
- contrary to ^^, this alignment style is robust in the sense that if you change the line with method name (e.g. add `public`) you don't have to move all the parameters, because they're simply indented, and not aligned with the opening `(`
- consistent (1 parameter per line)

Reply:
We really should take the time to publish the code style we're using. It's mostly the KIE code style, with some notable exceptions.